docs/collecting-data/collecting-from-own-applications/scala-tracker/initialization/index.md

```diff
@@ -0,0 +1,217 @@
+---
+title: "Initialization"
+date: "2022-9-15"
+sidebar_position: 10
+---
+
+Assuming you have completed the Scala Tracker Setup, you are ready to initialize the Scala Tracker.
+
+## Emitters
+
+Each tracker instance must be initialized with an Emitter which is responsible for firing events to a Collector. We offer two different modules of emitter: the http4s-emitter and the id-emitter
+
+### Http4s Emitter
+
+Backed by a [Http4s](https://http4s.org/) client, this emitter captures actions in the context of a functional effect type, such as a cats IO, ZIO or Monix Task. This is the recommended emitter if you are familiar with functional programming and the cats ecosystem of type classes.
+
+```scala
+import cats.data.NonEmptyList
+import cats.effect.std.Random
+import cats.effect.{IO, Resource}
+import org.http4s.ember.client.EmberClientBuilder
+
+import com.snowplowanalytics.snowplow.scalatracker._
+import com.snowplowanalytics.snowplow.scalatracker.Emitter._
+import com.snowplowanalytics.snowplow.scalatracker.emitters.http4s._
+
+val resource: Resource[IO, Tracker[IO]] = for {
+  client <- EmberClientBuilder.default[IO].build
+  rng <- Resource.eval(Random.scalaUtilRandom[IO])
+  emitter1 <- Http4sEmitter.build[IO](EndpointParams("mycollector.com"), client)(implicitly, rng)
+  emitter2 <- Http4sEmitter.build[IO](EndpointParams("myothercollector.com", port = Some(8080)), client)(implicitly, rng)
+} yield new Tracker(NonEmptyList.of(emitter1, emitter2), "mytrackername", "myapplicationid")
+
+resource.use { tracker =>
+  // Use the tracker inside this block to initialize and run your application
+  MyApp.run(tracker)
+}
+```
+
+The above code:
+
+- Creates a Http4s client from the EmberClientBuilder. Http4s offers other variants of the client, but Ember is the recommended default choice.
+- Creates a Random instance, required to randomize the delay on request retries
+- Creates an emitter `emitter1` which sends events to "mycollector.com" on the default port 80
+- Creates a second emitter `emitter2` which sends events to “myothercollector.com” on port 8080
+- Creates a tracker which can be used to send events to all emitters
+
+Your application might then use the tracker by flatMapping over the functional effect:
+
+```scala
+for {
+  _ <- tracker.trackPageView("http://example.com")
+  _ <- tracker.trackTransaction("order1234", 42.0)
+} yield ()
+```
+
+### ID Emitters
+
+The emitters in the "id" module have a simpler developer interface, because return types are not wrapped in functional events. Use these emitters if your application code is not using any other typeclasses from the cats ecosystem.
+
+The above code:
```

Comment:
Should it say the below code?

Reply:
Also, since it’s simpler, should it be the first to be explained? I haven’t been active in the scala community for a few years now, but I imagine there are still quite a few people who don’t care about monads and effects :)